api/views.py

```diff
@@ -394,7 +394,9 @@ def _validate(value):
         )
         study_group.save()
         generate_all_meetings(study_group)
-        return json_response(request, {"status": "created"});
+
+        study_group_url = "https://learningcircles.p2pu.org" + reverse('studygroups_signup', args=(slugify(study_group.venue_name), study_group.id,))
```

Comment:
This should use settings.DOMAIN, see https://github.com/p2pu/learning-circles/blob/master/learnwithpeople/settings.py#L185